sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -872,6 +873,38 @@ void _nib_ft_get(const _nib_offl_entry_t *dst, gnrc_ipv6_nib_ft_t *fte);
 int _nib_get_route(const ipv6_addr_t *dst, gnrc_pktsnip_t *ctx,
                    gnrc_ipv6_nib_ft_t *entry);
 
+#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_QUEUE_PKT)
+/**
+ * @brief Flush the packet queue of a on-link neighbor.
+ *
+ * @param node neighbor entry to be flushed
+ */
+void _nbr_flush_pktqueue(_nib_onl_entry_t *node);
+
+/**
+ * @brief Remove oldest packet from a on-link neighbor's packet queue.
+ *
+ * @param node neighbor entry
+ *
+ * @retval pointer to the packet entry or NULL if the queue is empty
+ */
+gnrc_pktqueue_t *_nbr_pop_pkt(_nib_onl_entry_t *node);
+
+/**
+ * @brief Push packet to a on-link neighbor's packet queue.
+ *
+ * @pre Neighbor is INCOMPLETE.
+ *
+ * @note If the queue size is @ref CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP,
+ *       this will @ref _nbr_pop_pkt() the oldest packet and release it.
+ *
+ * @param node neighbor entry
+ * @param pkt packet to be pushed
+ */
+void _nbr_push_pkt(_nib_onl_entry_t *node, gnrc_pktqueue_t *pkt);
+
+#endif
```

Comment:
```suggestion
#else
#define _nbr_flush_pktqueue(node) (void)node
#endif
```

allows you to drop some `#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_QUEUE_PKT)` in the code